templates/Rehydration.js

```diff
@@ -6,7 +6,6 @@ import DebugConfig from '../Config/DebugConfig'
 
 const updateReducers = (store: Object) => {
   const reducerVersion = ReduxPersist.reducerVersion
-  const config = ReduxPersist.storeConfig
```

Comment:
Can you explain this change? Do we not need to apply the config here now?

Reply:
Nevermind. I just checked out the migration guide https://github.com/rt2zz/redux-persist#migration-from-v4-to-v5